src/main/java/net/minecraftforge/fmp/microblock/IMicroblockPlacementGrid.java

```diff
@@ -0,0 +1,22 @@
+package net.minecraftforge.fmp.microblock;
+
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+/**
+ * Represents a grid for {@link IMicroblock} placement.
+ *
+ * @see IMicroblock
+ * @see MicroblockClass
+ */
+public interface IMicroblockPlacementGrid
```

Comment:
Why not just sideonly the class? It's a client side, where is this used need more information.